### ISSUE TITLE
Analyzing the graphs

### ISSUE THREAD

**Issue by kseignon** [kseignon/sds192-mp2#2]
Hi Guys!
Let's discuss the graphs in class tomorrow. I will see you then

**Comment by nashshaba** [comment]
Yes I've read through the drafts already and will work on editing the write-up in class
